File: Laboratorio hackeramk/01 Python/03_encapsulamiento.py
class Cuenta:
    def __init__(self, saldo):
        self.__saldo = saldo

    def depositar(self, cantidad):
        self.__saldo += cantidad

    def mostrar_saldo(self):
        print("Saldo:", self.__saldo)

cuenta = Cuenta(100)
cuenta.depositar(50)
cuenta.mostrar_saldo()

File: Laboratorio hackeramk/04 C++/03_encapsulamiento.cpp
#include <iostream>
using namespace std;

class Cuenta {
private:
    double saldo;

public:
    Cuenta(double s) {
        saldo = s;
    }

    void depositar(double cantidad) {
        saldo += cantidad;
    }

    void mostrarSaldo() {
        cout << "Saldo: " << saldo << endl;
    }
};

int main() {
    Cuenta cuenta(100);
    cuenta.depositar(50);
    cuenta.mostrarSaldo();
    return 0;
}
